.github/workflows/pr-commitlint.yml

```diff
@@ -36,11 +38,13 @@ jobs:
             # Ensure code-review commits don't get merged
             sed "s/code-review-rule': \[0/code-review-rule': [2/g" -i commitlint.config.js
             npx commitlint --from $first_commit --to $last_commit -V
-
             git log --pretty=format:%s $first_commit..$last_commit > ./subjects
             duplicates="$(cat ./subjects | sort | uniq -D)"
-            if [ "$duplicates" != "" ]; then
-              echo -e "Duplicate commits found:\n$duplicates" >&2
-              exit 1
+            ignore_dups="$(gh pr view ${{ github.event.pull_request.number }} --json labels -q .labels.[].name | grep -w 'ignore duplicate commits' || true)"
```

Comment:
I am unaware of the conventions here, but I assume the label name here is `ignore duplicate commits`. Would it be prudent to keep label as single word without spaces, like in general labels, `ignore_duplicate_commits`